backend/src/test/java/com/google/sps/EmailNotifierTest.java

```diff
@@ -0,0 +1,168 @@
+package com.google.sps;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.google.api.services.gmail.Gmail;
+import com.google.api.services.gmail.model.Message;
+import com.google.common.truth.Truth;
```

Comment:
optional: this can be imported statically (to remove the Truth. prefix to all the method calls) just as the mockito methods are above

Reply:
Imported statically